### PR TITLE
bsp: imx: enable CONFIG_GPIO_MXC

### PR DESCRIPTION
**Pull request by vanmaegima**
This config is missing in kernel 5.10+, so explicit enable it for all supported imx boards.

Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>

### CODE REVIEW

**Review by ricardosalveti — APPROVED** [review]
LGTM (once merged please also push to main/master).

**Review by MrCry0 — APPROVED** [review]

**Pull request merged**
